app/api/groups/groups.go

```diff
@@ -79,3 +81,34 @@ func checkThatUserHasRightsForDirectRelation(store *database.DataStore, user *da
 	}
 	return service.NoError
 }
+
+func (srv *Service) acceptOrRejectRequests(w http.ResponseWriter, r *http.Request,
+	action database.GroupGroupTransitionAction) service.APIError {
```

Comment:
Most GroupGroupTransitionAction values are invalid and the type is defined in another package. I would use an enumeration specific to this package with only "accept" and "reject" as values... it also means you need to create a method on it to generate the matching database.GroupGroupTransitionAction value... hum 🤔... bit longer but cleaner.

Reply:
fixed